cmd/querygen/main.go

```diff
@@ -54,5 +54,9 @@ func main() {
 			Title: "Build PipelineRun started events",
 			Query: querygen.GenBuildPipelineRunStartedQuery(*index),
 		},
+		{
+			Title: "Build TaskRun Completed event",
```

Comment:
Isn't the event about a `clair-scan`?  is that the only type of TaskRun?

Reply:
```suggestion
			Title: "Clair scan TaskRun completion events",
```